images/linux/scripts/installers/google-chrome.sh

```diff
@@ -9,7 +9,7 @@ LSB_RELEASE=$(lsb_release -rs)
 wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
 echo "deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main" | sudo tee /etc/apt/sources.list.d/google-chrome.list
 apt-get update
-apt-get install -y google-chrome-stable
+apt-get install -y google-chrome-stable fonts-noto-color-emoji
```

Comment:
I think this package should be moved to toolset files: https://github.com/actions/virtual-environments/blob/main/images/linux/toolsets/toolset-2004.json